openapi-v2.yaml

```diff
@@ -2684,53 +2667,14 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
-  /groups/{group_namespace}/{group_name}/content_activity:
+  /groups/{group_namespace}/{group_name}/total_activity:
```

Comment:
nit: could this be just activity? I think content_activity implies a subset, but activity itself should suffice without getting more specific